src/main/java/org/apache/commons/math4/stat/descriptive/moment/Variance.java

```diff
@@ -418,11 +418,10 @@ public double evaluate(final double[] values, final double mean,
                     accum += dev * dev;
                     accum2 += dev;
                 }
-                double len = length;
```

Comment:
`len` is a double but `length` is an int, I suspect it's preferable to cast only once here.

Reply:
Hi @ebourg 
sounds fair. Changed
TY